### ISSUE TITLE
tests.networking.scripted.macvlan.x86_64-linux blocking nixos-unstable

### ISSUE THREAD

**Issue by srhb** [NixOS/nixpkgs#93704]
**Describe the bug**

https://hydra.nixos.org/build/124277976

Bisect indicates e15815e885db42e642ed67527df76bbcc35f3e91 as the likely point of breakage.

cc @basvandijk 

**To Reproduce**

`nix-build nixos/release.nix -A tests.networking.scripted.macvlan.x86_64-linux`

**Comment by doronbehar** [comment]
I can't reproduce the error hydra has, it fails earlier for me on current master:

```
these derivations will be built:
  /nix/store/nsn8akgcr3hzfv6qv7gk53vfxgaxyyyq-etc.drv
  /nix/store/i1lpxng96hyb726q748s5sv5amvvklf5-nixos-system-client-20.09.git.f7b0b3c7002.drv
  /nix/store/3c3yby6zwvmabl5iwnsfhf5bzzzsxpx8-closure-info.drv
  /nix/store/b9zv35zxmnif5na8lfqr1r3769d87qsz-run-nixos-vm.drv
  /nix/store/ms3ypkxri6nfwm0yzfs7f2qn3izjigi9-nixos-vm.drv
  /nix/store/n16b58asjm4vgxfqdb07q07vhy18pdmm-etc.drv
  /nix/store/c07gpgx2x7n0ic2a0cw350g6lgj56d0m-nixos-system-router-20.09.git.f7b0b3c7002.drv
  /nix/store/cb4aypg0ygbbvadp7g42ffraikkwrr0a-closure-info.drv
  /nix/store/rfl4syjrswhykk9w6zxkxmr8nz2zqm7m-run-nixos-vm.drv
  /nix/store/zga0ir2a6rjqhzb21r3c9bwma918mc8c-nixos-vm.drv
  /nix/store/1cmvqmbkzf3pa98s3n7w3bbgw66yxlhy-nixos-test-driver-MACVLAN-Networking-Scripted.drv
  /nix/store/jp0vx04h23wyz9r8yi6wncfj9msnnvw0-vm-test-run-MACVLAN-Networking-Scripted.drv
building '/nix/store/n16b58asjm4vgxfqdb07q07vhy18pdmm-etc.drv'...
building '/nix/store/nsn8akgcr3hzfv6qv7gk53vfxgaxyyyq-etc.drv'...
ln: failed to create symbolic link '/nix/store/92g6abrz8l674bhix4cz6gjsh9b4j29v-etc/etc/tmpfiles.d/lvm2.conf': Permission denied
builder for '/nix/store/n16b58asjm4vgxfqdb07q07vhy18pdmm-etc.drv' failed with exit code 1
cannot build derivation '/nix/store/c07gpgx2x7n0ic2a0cw350g6lgj56d0m-nixos-system-router-20.09.git.f7b0b3c7002.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/zga0ir2a6rjqhzb21r3c9bwma918mc8c-nixos-vm.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/1cmvqmbkzf3pa98s3n7w3bbgw66yxlhy-nixos-test-driver-MACVLAN-Networking-Scripted.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/jp0vx04h23wyz9r8yi6wncfj9msnnvw0-vm-test-run-MACVLAN-Networking-Scripted.drv': 1 dependencies couldn't be built
error: build of '/nix/store/jp0vx04h23wyz9r8yi6wncfj9msnnvw0-vm-test-run-MACVLAN-Networking-Scripted.drv' failed
```

**Comment by vcunat** [comment]
We've had a staging-next merge in the meantime; this symlink problem was fixed by #93838.

**Comment by charvp** [comment]
Local build works with my changes in #93893. Should this be tested via ofborg? If so how do I do this (can I even do this)?
